apps/web-portal/lib/use-cases/auth/AuthLogin.mjs

```diff
@@ -2,10 +2,28 @@ import {UseCase} from '@zorko-io/util-use-case'
 import {apiToken} from '../utils'
 
 export class AuthLogin extends UseCase {
-  // eslint-disable-next-line no-unused-vars
+  static rules = {
+    credentials: [{
+      or: [{
+        nested_object: {
+          password: ['required'],
+          email: ['required', 'email']
+        }
+      }, {
+        nested_object: {
+          token: ['required']
+        }
+      }]
+    }, 'required']
+  }
+
+
   async run(params) {
-    if (params.token) {
-      apiToken.verify(params.token)
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Could we avoid that eslint directive ?

Reply:
yes, but then we need to use all the 'token, password, email' or remove the next row
leave this directive temporary before we add logic related to the pwd and email